img/info.py

```diff
@@ -34,6 +37,7 @@
     f";TIME:{time_s:.2f}",
     f";Filament used:{filament_used_mm/1000:.5f}m",
     f";Layer height:{layer_height}",
+    "';Generated by EnderV3SEPrusaPostScript",
```

Comment:
Can you instead make this a link to this repo, [vaguely like the config bundle](https://github.com/suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config/blob/main/generate_config_bundle.py#L4)?